### PR TITLE
[CHA-530] Run grammarly on public doc

### PR DESCRIPTION
**Pull request by AdelELHAIBA-Akeneo**

### CODE REVIEW

**Review by guillaumefer-akeneo — APPROVED** [review]

**Review by JulienVerbrugge — APPROVED** [review]

**Comment by akeneo-secops** [comment]
### Gitleaks PR check
🟢 We detected no secrets within this PR! Good job 🎉

File: content/graphql/browse-graphql-capabilities.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ You can expand or collapse tree nodes to navigate the schema. Clicking on a type
 :::
 
 :::info
-Clicking on a field adds it to the query in the centre panel.
+Clicking on a field adds it to the query in the center panel.
```
Comment (yvesmotteux) [inline content/graphql/browse-graphql-capabilities.md:45]: :scream:

File: content/graphql/queries-and-arguments.md [hunk 2]
```diff
@@ -20,12 +20,12 @@ The following arguments are available:
 | `uuid`             | **String[]** | Get only specific items based on their uuid<br/>example `uuid: ["002844f9-a470-42e2-8268-ddfd8f646593", "002844f9-3648-8268-8268-ddfd8f6411545"]`                                     |
 | `parent`           | **String**   | Get **product variants** linked to the product models base on its **code**<br/>example `code: "code1"`                                                                                |
 | `noParent`         | **Boolean**  | Only accepted value `true`, will return **only simple products**.                                                                                                                     |
-| `attributesToLoad` | **String[]** | Restrict the attribute values we will load on product, more details on [Best practices](/graphql/best-practices.html#load-only-required-product-attributes)                           |
+| `attributesToLoad` | **String[]** | Restrict the attribute values loadded on the product, more details on [Best practices](/graphql/best-practices.html#load-only-required-product-attributes)                       |
```
Comment (yvesmotteux) [inline content/graphql/queries-and-arguments.md:23]: I think loaded only takes one d

File: content/graphql/queries-and-arguments.md [hunk 3]
```diff
@@ -20,12 +20,12 @@ The following arguments are available:
...
 
 :::info
-The `categories`, `families` and `uuid` use under the hood the `Rest API search capabilities` to ease the data fetching.
+The `categories`, `families`, and `uuid` are using under the hood of the `Rest API search capabilities` to ease the data fetching.
```
Comment (yvesmotteux) [inline content/graphql/queries-and-arguments.md:26]: This doesn't feel correct, maybe `The categories, families, and uuid queries are using the Rest API search capabilities under the hood to ease the data fetching.`

File: content/graphql/use-cases.md [hunk 4]
```diff
@@ -1,17 +1,17 @@
-# Use cases examples
+# Uses case examples
```
Comment (yvesmotteux) [inline content/graphql/use-cases.md:1]: I think it was correct before

File: content/graphql/use-cases.md [hunk 5]
```diff
@@ -1,17 +1,17 @@
...
 :::info
-We highly recommand to use `variables` to make your query dynamic.
+We highly recommend using `variables` to make your query dynamic.
 
 [More details](/graphql/best-practices.html#variables-usages)
 :::
 
 :::warning
-All bellow example are demo data, and you must adapt the queries regarding your `attributes codes`, `locales`, `product model code` ...
+All the below examples are demo data, and you must adapt the queries regarding your `attributes codes`, `locales`, `product model code` ...
```
Comment (yvesmotteux) [inline content/graphql/use-cases.md:14]: not sure about this one, but maybe switching "below" and "examples" will sound better?

**Pull request merged**